controllers/users-controller.js

```
@@ -875,6 +875,47 @@ const claimPromoCode = async (req, res, next) => {
   }
 };
 
+const verifySms = async (req, res, next) => {
+  // Validating User Inputs
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    return next(new ErrorHandler(422, 'Invalid verification code or phone'));
+  }
+
+  // Defining User Inputs
+  const { phone, smsToken, userId } = req.body;
+
+  const user = await userService.getUserById(userId);
+  if (!user) {
+    throw new Error('User not found, please try again', 422);
+  }
+  try {
+    await userService.verifySms(user, phone, smsToken);
+
+    res.status(200).send();
+  } catch (err) {
+    console.log(err);
+    next(new ErrorHandler(422, err.message));
```

Let's not return internal errors to the FE? I would rather return a generic error e.g. "Sms verification failed" with status 500.